website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js

```diff
@@ -51,4 +51,9 @@ app.controller('MainController',
         var cmdlet = productInfo[$routeParams.product][$routeParams.cmdlet];
         return cmdlet.parameters;
       };
+      
+      /* Tells us if the object passed in is empty */
```

Comment:
Do you mean to convert multi-line comments into series of single line comments? I'm not sure I understand.